src/pydvl/utils/numeric.py

```diff
@@ -111,24 +122,32 @@ def random_powerset(
         ValueError: if the element sampling probability is not in [0,1]
 
     """
+    if not isinstance(s, np.ndarray):
```

Comment:
We don't do runtime type checking anywhere. Why here?

Reply:
Removed.